Pipfile

```diff
@@ -16,6 +16,8 @@ colorama = "*"
 verboselogs = "*"
 crcmod = "*"
 tinyaes = "*"
+requests = "*"
```

Comment:
I'd like to avoid including new libraries for this, we can probably use the ones within the standard library